### ISSUE TITLE
Page fault when using menus in scaled X11 applications

### ISSUE THREAD

**Issue by eunichron** [pop-os/cosmic-epoch#1113]
Problem: Accessing menus in X11 apps with scaling turned on causes a page fault.

Snippet from journalctl:

```
Oct 12 07:36:51 localhost cosmic-comp[4140]: tiling exceptions custom config error: GetKey("tiling_exception_custom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
Oct 12 07:36:52 localhost cosmic-panel[4221]: com.system76.CosmicAppletStatusArea: Acquired bus name: org.kde.StatusNotifierWatcher
Oct 12 07:37:00 localhost kernel: BUG: unable to handle page fault for address: 0000000007f34050
Oct 12 07:37:00 localhost kernel: #PF: supervisor read access in kernel mode
Oct 12 07:37:00 localhost kernel: #PF: error_code(0x0000) - not-present page
-- Boot eec83751f87f43dabd27a92085b9c854 --
```

Results in an unrecoverable hard system freeze, forcing a reboot. It's happened to me with three different applications: MusicBrainz Picard, VLC, MKVToolNix (GUI). Turning off X11 application scaling in Settings is working as a temporary solution. 

Menus in those applications are also not being drawn correctly with scaling turned off- screenshot of VLC with off-axis menu 
![screenshot-2024-10-12-13-10-57](https://github.com/user-attachments/assets/01e5e597-b1b8-4e7a-9d13-8ccef275da64)
attached.

OS: Arch
Kernel: 6.11.3
COSMIC Package: AUR-git

**Comment by KodeBarista** [comment]
Are you still facing this issue? On Fedora with scaling it works as expected.

**Comment by eunichron** [comment]
> Are you still facing this issue? On Fedora with scaling it works as expected.

No, I haven't experienced this issue since one of the Alpha builds. It appears to have been fixed.

Completely forgot about this bug report. I'll close it.
